server.go

```diff
@@ -3311,3 +3312,49 @@ func copyURL(u *url.URL) *url.URL {
 	*other = *u
 	return other
 }
+
+func (s *Server) StartReportingLoop(version string, clusterID uint64) chan struct{} {
+	s.reportStats(version, clusterID)
+
+	ticker := time.NewTicker(24 * time.Hour)
+	for {
+		select {
+		case <-ticker.C:
+			s.reportStats(version, clusterID)
+		}
+	}
+}
+
+func (s *Server) reportStats(version string, clusterID uint64) {
+	json := fmt.Sprintf(`[{
+    "name":"reports",
+    "columns":["os", "arch", "version", "id", "cluster_id", "num_series", "num_measurements", "num_databases"],
+    "points":[["%s", "%s", "%s", "%x", "%x", "%d", "%d", "%d"]]
+  }]`, runtime.GOOS, runtime.GOARCH, version, s.ID(), clusterID, s.numSeries(), s.numMeasurements(), len(s.databases))
+
+	data := bytes.NewBufferString(json)
+
+	log.Printf("Sending anonymous usage statistics to m.influxdb.com")
+	client := http.Client{Timeout: time.Duration(5 * time.Second)}
+	go client.Post("http://m.influxdb.com:8086/db/reporting-test/series?u=reporter&p=influxdb", "application/json", data)
+}
+
+func (s *Server) numSeries() int {
```

Comment:
In that case I would just move the code out of those functions, remove the functions, and just run the calculations in `reportStats`. `server.go` is getting big enough as it is. This is just a style suggestion of course.